tests/commonTemplates_test.go

```diff
@@ -431,7 +431,196 @@ var _ = Describe("Common templates", func() {
+				table.Entry("[test_id:4775]: ServiceAccounts with view role can view PVCs",
+					&authv1.SubjectAccessReviewSpec{
+						User:   regularSAFullName,
+						Groups: sasGroup,
+						ResourceAttributes: &authv1.ResourceAttributes{
+							Verb:      "get",
+							Namespace: commonTemplates.GoldenImagesNSname,
+							Resource:  "persistentvolumeclaims",
+						},
 					}))
+			table.DescribeTable("regular service account RBAC", expectUserCannot,
+				table.Entry("[test_id:4776]: ServiceAccounts with only view role cannot create PVCs",
+					&authv1.SubjectAccessReviewSpec{
+						User:   regularSAFullName,
+						Groups: sasGroup,
+						ResourceAttributes: &authv1.ResourceAttributes{
+							Verb:      "create",
+							Namespace: commonTemplates.GoldenImagesNSname,
+							Resource:  "persistentvolumeclaims",
+						},
+					}),
+				table.Entry("[test_id:4846]: ServiceAccounts with only view role cannot delete PVCs",
+					&authv1.SubjectAccessReviewSpec{
+						User:   regularSAFullName,
+						Groups: sasGroup,
+						ResourceAttributes: &authv1.ResourceAttributes{
+							Verb:      "delete",
+							Namespace: commonTemplates.GoldenImagesNSname,
+							Resource:  "persistentvolumeclaims",
+						},
+					}),
+				table.Entry("[test_id:4879]: ServiceAccounts with only view role cannot create any other resources other than the ones listed in the View role",
+					&authv1.SubjectAccessReviewSpec{
+						User:   regularSAFullName,
+						Groups: sasGroup,
+						ResourceAttributes: &authv1.ResourceAttributes{
+							Verb:      "create",
+							Namespace: commonTemplates.GoldenImagesNSname,
+							Resource:  "pods",
+						},
+					}),
+			)
+			Context("With Edit permission", func() {
+				BeforeEach(func() {
+					regularSA = &core.ServiceAccount{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      "regular-sa",
+							Namespace: strategy.GetNamespace(),
+						},
+					}
+					Expect(apiClient.Get(ctx, getResourceKey(regularSA), regularSA)).ToNot(HaveOccurred())
+
+					editObj := &rbac.RoleBinding{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      "test-edit",
+							Namespace: commonTemplates.GoldenImagesNSname,
+						},
+						Subjects: []rbac.Subject{
+							{
+								Kind:      "ServiceAccount",
+								Name:      regularSA.GetName(),
+								Namespace: strategy.GetNamespace(),
+							},
+						},
+						RoleRef: rbac.RoleRef{
+							Kind:     "ClusterRole",
+							Name:     commonTemplates.EditClusterRoleName,
+							APIGroup: "rbac.authorization.k8s.io",
+						},
+					}
+					Expect(apiClient.Create(ctx, editObj)).ToNot(HaveOccurred(), "Failed to create RoleBinding")
+				})
+				table.DescribeTable("should verify resource permissions", expectUserCan,
+					table.Entry("[test_id:4774]: ServiceAcounts with edit role can create PVCs",
+						&authv1.SubjectAccessReviewSpec{
+							User:   "system:serviceaccount:ssp-operator-functests:regular-sa",
```

Comment:
Shouldn't this be `regularSAFullName`?

Reply:
I'm seeing that the nested `Context ` block is somehow making the value of `regularSAFullName` to be `""`
Probably a bug